scripts/windows/auto-config.ps1

```diff
@@ -0,0 +1,122 @@
+#Requires -RunAsAdministrator
+
+# !!! This script is not meant to be run without administrator rights. !!!
+
+$env_directory = "../env"
```

Comment:
I would change paths to relative paths instead of '../'. Like:
$workdir = Get-Location
$env_directory = "$workdir/env"